airbyte-ci/connectors/CONNECTOR_CHECKLIST.yaml

```diff
@@ -0,0 +1,12 @@
+paths:
+  "airbyte-integrations/connectors/**":
+    - PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
+    - "[Breaking changes are considered](https://docs.airbyte.com/contributing-to-airbyte/#breaking-changes-to-connectors). If a **Breaking Change** is being introduced, ensure an Airbyte engineer has created a Breaking Change Plan and you've followed all steps in the [Breaking Changes Checklist](https://docs.airbyte.com/contributing-to-airbyte/#checklist-for-contributors)"
```

Comment:
https://docs.airbyte.com/contributing-to-airbyte/#breaking-changes-to-connectors not going to the correct place, can you double check it?

Reply:
Same for https://docs.airbyte.com/contributing-to-airbyte/#checklist-for-contributors maybe another PR is update the docs?